collector/zookeeper/collector.yml

```diff
@@ -0,0 +1,32 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+      grpc:
+  prometheus/zookeeper:
+    config:
+      scrape_configs:
+        - job_name: otel-zookeeper-eg
+          scrape_interval: 5s
+          static_configs:
+            - targets: ["zookeeper:7000"]
+
+exporters:
+  logging:
+    loglevel: debug
+  otlp/public:
+    endpoint: ingest.lightstep.com:443
+    headers:
+        "lightstep-access-token": "${LS_ACCESS_TOKEN}"
+  file:
```

Comment:
Is this something we have been using in other examples? Otherwise, is there a reason to be using this here specifically?